workspaces/rbac/plugins/rbac/src/components/CreateRole/AddMembersForm.tsx

```diff
@@ -89,22 +133,25 @@ export const AddMembersForm = ({
     setSelectedMember(selectedMembers);
   }, [selectedMembers]);
 
+  const [dialogOpen, setDialogOpen] = useState(false);
+
+  const { users, groups } = useMemo(() => {
+    const u: UserEntity[] = [];
+    const g: GroupEntity[] = [];
```

Comment:
Optional nit: much prefer not using single letter variables, I know we use `t` but that's a convention of the translation pattern.